src/components/JapaneseText.tsx

```
@@ -0,0 +1,23 @@
+
+const JapaneseText: React.FC<JapaneseTextProps> = ({ text }) => {
+  // Split the text by the bold pattern and map each part
+  const parts = text.split(/(\*\*+[^*]+\*\*+)/g);
+  return (
+    <div>
+      {parts.map((part, index) => {
+        if (part.startsWith('**') && part.endsWith('**')) {
+          const boldText = part.substring(2, part.length - 2);
+          return <b key={index}>{boldText}</b>;
+        }
+        return <>{part}</>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)